app/models/book.rb

```diff
@@ -1,3 +1,23 @@
 class Book < ApplicationRecord
-  
+  has_many :reviews
+
+  def sum_reviews
+    reviews.sum(:rating)
+  end
+
+  def count_reviews
+    reviews.count
+  end
+
+  def average_rating
+    sum_reviews/count_reviews
```

Comment:
`reviews.average(:rating)` and this would cut out two other methods `sum_reviews` and `count_reviews`